integration_tests/test_basic.py

```diff
@@ -277,8 +277,29 @@ def test_merge_participant(apps, as_admin):
     success_alert = page.locator("div.alert.alert-success")
     expect(success_alert).to_have_text("Participant merged!")
     assert success_alert.is_visible()
-
     page.goto(f"{apps.backend.url}/participants/")
     table_body = page.locator("#DataTables_Table_0 tbody")
     rows_count = table_body.locator("tr").count()
-    assert rows_count == 1, f"Expected 1 participant, but found {rows_count}"
\ No newline at end of file
+    assert rows_count == 1, f"Expected 1 participant, but found {rows_count}"
+
+def test_comment(apps,as_admin):
+
+    """ Test if comments can be placed and are visible """
+
+    Experiment = apps.backend.get_model('experiments', 'Experiment')
```

Comment:
I've missed it in the other PRs, but there are several places where `Experiment` is defined but not used